inst/scripts/make-blueprint_encode-metadata.R

```diff
@@ -0,0 +1,27 @@
+write.csv(file="../extdata/metadata-blueprint_encode.csv", 
+          data.frame(
+              Title = sprintf("Blueprint/Encode RNA-seq data: %s", c("normcounts", "colData")), # This can be the exact file name (if self-describing) or a more complete description.
```

Comment:
If you'll forgive the nitpicking:

- I generally like the `Title` to be grammatically simple, so no colons; something like `Blueprint/ENCODE RNA-seq normcounts` and `Blueprint/ENCODE RNA-seq colData`.
- If you're spelling things out anyway, `norm.`  should be `normalized`.

Reply:
lol, you do you